sci-libs/fftw/fftw-3.3.5.ebuild

```diff
@@ -0,0 +1,180 @@
+			doc/fftw3."info" || die "failed to fix info file"
+
+		eautoreconf
+	fi
+}
+
+multilib_src_configure() {
+	# there is no abi_x86_32 port of virtual/mpi right now
+	local enable_mpi=$(use_enable mpi)
+	multilib_is_native_abi || enable_mpi="--disable-mpi"
+
+	# jlec reported USE=quad on abi_x86_32 has too few registers
+	# stub Makefiles
+	if [[ ${MULTILIB_ABI_FLAG} == abi_x86_32 && ${MULTIBUILD_ID} == quad-* ]]; then
+		mkdir -p "${BUILD_DIR}/tests" || die
+		echo "all: ;" > "${BUILD_DIR}/Makefile" || die
+		echo "install: ;" >> "${BUILD_DIR}/Makefile" || die
+		echo "smallcheck: ;" > "${BUILD_DIR}/tests/Makefile" || die
+		return 0
+	fi
+
+	local myconf=(
+		$(use_enable "cpu_flags_x86_fma$(usex cpu_flags_x86_fma3 3 4)" fma)
+		$(use_enable fortran)
+		$(use_enable zbus mips-zbus-timer)
+		$(use_enable threads)
+		$(use_enable openmp)
+	)
+	case "${MULTIBUILD_ID}" in
+		single-*)
+			#altivec, sse, single-paired only work for single
+			myconf+=(
+				--enable-single
+				$(use_enable altivec)
+				$(use_enable cpu_flags_x86_avx avx)
+				$(use_enable cpu_flags_x86_sse sse)
+				${enable_mpi}
+				$(use_enable neon)
+			)
+			;;
+
+		double-*)
+			myconf+=(
+				$(use_enable cpu_flags_x86_avx avx)
+				$(use_enable cpu_flags_x86_sse2 sse2)
+				${enable_mpi}
+			)
+			;;
+
+		longdouble-*)
+			myconf+=(
+				--enable-long-double
+				${enable_mpi}
+			)
+			;;
+
+		quad-*)
+			#quad does not support mpi
+			myconf+=( --enable-quad-precision )
+			;;
+
+		*)
+			die "${MULTIBUILD_ID} precision not implemented in this ebuild"
+			;;
+	esac
+
+	ECONF_SOURCE="${S}" econf "${myconf[@]}"
+}
+
+src_configure() {
+	# filter -Os according to docs
+	replace-flags -Os -O2
+
+	multibuild_foreach_variant multilib-minimal_src_configure
+}
+
+src_compile() {
+	multibuild_foreach_variant multilib-minimal_src_compile
+}
+
+multilib_src_test() {
+	emake -C tests smallcheck
+}
+
+src_test() {
+	# We want this to be a reasonably quick test, but that is still hard...
+	ewarn "This test series will take 30 minutes on a modern 2.5Ghz machine"
+	# Do not increase the number of threads, it will not help your performance
+	# local testbase="perl check.pl --nthreads=1 --estimate"
+	#     ${testbase} -${p}d || die "Failure: $n"
+
+	multibuild_foreach_variant multilib-minimal_src_test
+}
+
+src_install() {
+	DOCS=( AUTHORS ChangeLog NEWS README TODO COPYRIGHT CONVENTIONS )
+	HTML_DOCS=( doc/html/ )
+
+	multibuild_foreach_variant multilib-minimal_src_install
+
+	if use doc; then
+		dodoc doc/*.pdf
+		docinto faq
+		dodoc -r doc/FAQ/fftw-faq.html/*
+	else
+		rm -r "${ED}"/usr/share/doc/${PF}/html || die
+	fi
+
+	local x
+	for x in "${ED}"/usr/lib*/pkgconfig/*.pc; do
+		local u
+		for u in $(usev mpi) $(usev threads) $(usex openmp omp ""); do
+			sed -e "s|-lfftw3[flq]\?|&_${u} &|" "$x" > "${x%.pc}_${u}.pc" || die
+		done
+	done
+
+	# fftw uses pkg-config to record its private dependencies
+	find "${ED}" -name '*.la' -delete || die
```

Comment:
Use `prune_libtool_files()` from `eutils.eclass` instead, it is a bit more verbose in the output.